src/app/components/Timestamp/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node } from 'prop-types';
+import { T_BREVIER } from '../../lib/constants/typography';
+import {
+  C_RHINO,
+  GEL_SPACING_DBL,
+  FF_NEWS_SANS_REG,
+} from '../../lib/constants/styles';
+
+const StyledTimestamp = styled.span`
+  ${T_BREVIER};
+  color: ${C_RHINO};
+  font-family: ${FF_NEWS_SANS_REG};
+  padding-bottom: ${GEL_SPACING_DBL};
+`;
+
+const Timestamp = ({ children }) => (
```

Comment:
This component could also export the `time` element.

It could also do all of the logic that the container component is doing, and we just pass it a single timestamp props. There doesn't seem much value in separating the logic as it is fairly simple and of similar concern.